neverbleed.c

```diff
@@ -965,7 +1009,10 @@ static EVP_PKEY *ecdsa_create_pkey(neverbleed_t *nb, size_t key_index, int curve
     exdata->key_index = key_index;
 
     ec_key = EC_KEY_new_method(nb->engine);
-    EC_KEY_set_ex_data(ec_key, 0, exdata);
+    NEVERBLEED_MULTITHREAD_ONCE({
+        exdata_idx = EC_KEY_get_ex_new_index(0, NULL, NULL, NULL, exdata_free_callback);
```

Comment:
`RSA_get_ex_new_index` and `EC_KEY_get_ex_new_index` reserves slots for different types of objects. Therefore I believe we should not be sharing the same variable (`exdata_idx`) for holding the two values?

In practice, I think our users will not be affected by the problem though, as neverbleed would likely be the only one that's using an exdata for either of the types and hence `exdata_idx` will always be one.